lib/data/external/AwsClient.js

```diff
@@ -355,6 +357,35 @@ class AwsClient {
             return callback();
         });
     }
+    copyObject(request, objectGetInfo, log, callback) {
+        // for backwards compatibility
+        const sourceObjectKey =
+        typeof(objectGetInfo) === 'string' ? objectGetInfo :
+          objectGetInfo.key;
```

Comment:
no need for this because if it's aws to aws or azure to azure it would not be the old format